syntax = "proto3";

package proto;

service FlightsInfo {
    rpc AddFlight(NewFlight) returns (AddResponse);
    rpc ModifyFlight(UpdatedFlight) returns (ModFlightResponse);
}

message NewFlight {
    string id = 1;
    string date = 2;
    string departureAirport = 3;
    string arrivalAirport = 4;
    string departureTime = 5;
    string arrivalTime = 6;
    string airline = 7;
    string price = 8;
    int32 seats = 9;
}

message AddResponse {
    bool isOk = 1;
}

message UpdatedFlight {
    string flightId = 1;
    string newPrice = 2;
}

message ModFlightResponse {
    bool isOk = 1;
}
